src-react/actions/bootstrap.js

```diff
@@ -0,0 +1,12 @@
+import { fetchUngitConfig } from './ungit-config';
+import { fetchLatestVersion, fetchGitVersion } from './version';
+import { pending } from './common';
+
+export function bootstrap() {
+  return dispatch => {
+    dispatch(pending(4));
```

Comment:
Due to ungitconfig action will also fetch userconfig, but after I check the code, there's a condition  to decide if fetching userconfig or not, so I think I need to move a pending call into ungitconfig

It's fixed